internal/start_command.go

```diff
@@ -15,5 +16,18 @@ func StartCommand(command []string) (waitFunc func() error, stdout, stderr io.Re
 	err = c.Start()
 	waitFuncResult := c.Wait
 	tracelog.ErrorLogger.FatalOnError(err)
-	return waitFuncResult, stdoutResult, stderrResult
+	return func() {
```

Comment:
here we lose code readability. I think this function is too big to declare it and call it in one place of the code

Reply:
I don't know other ways to declare this function with variables from StartCommand